docs/source/conf.py

```diff
@@ -266,7 +266,7 @@ def __setattr__(self, name, value):
 ]
 
 intersphinx_mapping = {'python': ('https://docs.python.org/3/', None),
-                       'paramiko': ('https://docs.paramiko.org/en/2.1/', None)}
+                       'paramiko': ('https://docs.paramiko.org/en/stable/', None)}
```

Comment:
just noticed that the docs were for some reason locked to a very old version of paramiko which did not have the `disable_algorithms` keyword argument. Since pwntools doesn't have any particular upper bound version for paramiko, I think it would make more sense if this was linked to stable since that will be what is most likely used? Alternatively, the lowest version of paramiko that does contain the `disable_algorithms` keyword arg is 2.6